llm_bench/python/utils/conversion_utils/helpers.py

```diff
@@ -77,12 +82,60 @@ def get_fp_path(args, model_subpath):
     return None
 
 
-def save_tokenizer(tokenizer, out_dir):
+# TODO: Replace with function from optimum-intel
+def save_ov_tokenizer(
+    tokenizer,
+    output_path: Union[str, Path],
+) -> None:
+    from transformers import T5Tokenizer, T5TokenizerFast
+
+    UNSUPPORTED_TOKENZIER_CLASSES = (
+        T5Tokenizer,
+        T5TokenizerFast,
+    )
+    if isinstance(tokenizer, UNSUPPORTED_TOKENZIER_CLASSES):
+        log.info("OpenVINO Tokenizer for this model is not supported.")
+        return
+
+    try:
+        from openvino_tokenizers import convert_tokenizer
+    except ModuleNotFoundError:
+        log.info("Run `pip install openvino-tokenizers` to get OpenVINO tokenizer/detokenizer models.")
+
+    if not isinstance(output_path, Path):
+        output_path = Path(output_path)
+
+    try:
+        converted = convert_tokenizer(tokenizer, with_detokenizer=True)
+    except NotImplementedError:
+        log.info("Detokenizer is not supported, convert tokenizer only.")
+        converted = convert_tokenizer(tokenizer, with_detokenizer=False)
+    except OVTypeError:
+        log.info("OpenVINO Tokenizer for this model is not supported.")
+        return
+    except Exception as exception:
+        log.warning(f"OpenVINO Tokenizer for this model is not supported. Exception: {exception}")
+        return
+
+    if not isinstance(converted, tuple):
+        converted = (converted,)
+
```

Comment:
if I right understnad it is not tuple in case if conversion with detokenizer files only? Maybe it make sense to move tuple creation under place where you convert tokenizer